snippets/csharp/System.Linq/Queryable/AggregateTSource/queryable.cs

```diff
@@ -695,14 +698,15 @@ static void DistinctEx1()
         static void ElementAtEx1()
         {
             // <Snippet28>
-            string[] names = { "Hartono, Tommy", "Adams, Terry",
+            string[] names = [ "Hartono, Tommy", "Adams, Terry",
                                "Andersen, Henriette Thaulow",
-                               "Hedlund, Magnus", "Ito, Shu" };
+                               "Hedlund, Magnus", "Ito, Shu" ];
 
-            Random random = new Random(DateTime.Now.Millisecond);
+            Random random = new(DateTime.Now.Millisecond);
 
             string name =
-                names.AsQueryable().ElementAt(random.Next(0, names.Length));
+                names.ElementAt(random.Next(0, names.Length));
+            //names[random.Next(0, names.Length) ];      // simpler
```

Comment:
Remove this. Yes, it's simpler, but only because the source is an array (or any other type with an index operator)